CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ### Packaging
 
 * MapboxCoreNavigation now requires [MapboxNavigationNative v123._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/123.1.0). ([#4298](https://github.com/mapbox/mapbox-navigation-ios/pull/4298))
-* MapboxNavigation now requires [MapboxMaps v10.10.0](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.10.0). ([#4298](https://github.com/mapbox/mapbox-navigation-ios/pull/4298))
+* MapboxNavigation now requires [MapboxMaps v10.10._x_](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.10.1). ([#4315](https://github.com/mapbox/mapbox-navigation-ios/pull/4315))
```

Comment:
Should we also mention the fix itself in the changelog?

Reply:
I think so, thanks for the suggestion, I'll add the fix to the ChangeLog.